science/libsbml/Portfile

```diff
@@ -1,12 +1,13 @@
 # -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
 
 PortSystem          1.0
+PortGroup           cmake 1.1
 
 name                libsbml
-version             5.13.0
+version             5.16.0
 categories          science
 platforms           darwin
-maintainers         nomaintainer
+maintainers         @funasoul gmail.com:funasoul openmaintainer
```

Comment:
Sorry to keep picking at details. This should be 

`maintainers         {@funasoul gmail.com:funasoul} openmaintainer`

And thank you for your patience.

Reply:
Thanks for pointing out the issue. I've committed the fix.